README.rst

```diff
@@ -38,22 +37,21 @@ Installation
 
 Install via ``pip``:
 
-.. code-block:: bash
+::
 
     $ pip install dropbox
 
-
 Install from source:
 
-.. code-block:: bash
+::
 
     $ git clone git://github.com/dropbox/dropbox-sdk-python.git
     $ cd dropbox-sdk-python
     $ python setup.py install
 
 After installation, to get started, open a Python console:
 
-.. code-block:: python
+::
```

Comment:
Does it auto detect the language? I thought giving it a suggestion improved the syntax highlighting.